python/CMakeLists.txt

```
@@ -2,10 +2,10 @@ cmake_minimum_required(VERSION 3.16)
 
 PROJECT(dolfinx_pybind11)
 
+find_package(Python COMPONENTS Interpreter Development REQUIRED)
```

```suggestion
find_package(Python3 COMPONENTS Interpreter Development REQUIRED)
```
I think this should be `Python3` to avoid falling back to Python2 (which we don't support).

---

Right, I've amended that. Thanks!